src/reducers/content/meta.js

```diff
@@ -5,6 +5,9 @@ export default (state = {}, action = {}) => {
 
   if (type === t.FETCH_DATA_SUCCESS && payload.dataType === 'content') {
     return payload.data.meta;
+
+  } else if (type === t.RESET_SESSION) {
```

Comment:
Better to use an `initialState = {}`, so we don't need to redefine it in two places when it changes